### ISSUE TITLE
Feature Request - Multiselect when organizing connections

### ISSUE THREAD

**Issue by Zalminen** [mRemoteNG/mRemoteNG#945]
I imported a bunch of servers from AD. Now as far as I can tell the only way to move the connections to the correct folders is by dragging them one by one which is slow when there's a lot of them.

Being able to select multiple connections by shift-clicking or ctrl-clicking and then dragging them all to the correct folder would make this kind of operation much faster. 

In addition to move by dragging it would also be nice if you could use cut & paste to move connections between the folders as mouse dragging isn't often as exact.

**Comment by havalo-lolawo** [comment]
@sparerd - Is this feature that's being considered because it would be nice to have instead of relying on Notepad++ to edit the XML file manually?

**Comment by sparerd** [comment]
Yes, we have been thinking about how to add mass-edit features for a while. Unfortunately there are a lot of other features ahead of it in the queue.
